docs/editing-components.md

```diff
@@ -421,6 +441,40 @@ In other words, if an existing field or block is changing,
 any data stored in that field or block has to be migrated to a different place,
 unless you're OK with jettisoning it.
 
+##### Not sure if there's actually any data that could potentially be lost?
+
+You may not know off the top of your head if a component you are modifying
+actually has any data stored that could be lost.
+One way that you can manually check to see if this is the case
+is to use the Block Inventory feature that
+[we added to Wagtail](https://github.com/cfpb/wagtail-inventory).
+This feature lets you search all Wagtail pages on the site
+for the presence of a component. Here's how:
+
+1. With a current database dump, visit http://localhost:8000/admin/inventory/.
+   (This is also available from the admin menu by going to
+   **Settings** > **Block Inventory**.)
+1. Using the three pairs of dropdown menus at the top of the page,
+   choose to look for pages that either
+   _include_ or _exclude_ particular components.
+    - If multiple components are selected, resulting pages must
+      match _all_ of the conditions.
+1. Click **Find matching pages** to execute the search.
+1. If no results are found, lucky you!
```

Comment:
What about preserving legacy uses of blocks on non-production systems, e.g. build?

Reply:
I do not want to encourage anyone to think of stuff on Build as permanent.